Update installing-stencil.md (#1317)
bigcommerce/dev-docs: docs/stencil-docs/installing-stencil-cli/installing-stencil.md

# Install Stencil CLI
npm install -g @bigcommerce/stencil-cli
```

> ### Installing on Mac with M1 chip

Reviewer: ```suggestion
### Installing on Mac with Apple silicone

```

> To install Stencil CLI and its dependencies on Mac with a M1 chip, open a Rosetta terminal and run the following commands.

Reviewer: ```suggestion
Installing Stencil CLI and its dependencies on Macs that use Apple silicone, such as the M1 chip, requires Rosetta. Rosetta allows a Mac with Apple silicon to use apps built for a Mac with an Intel processor. The following steps will guide you through the installation process:

```

<div class="HubBlock--callout">
<div class="CalloutBlock--info">
<div class="HubBlock-content">

<!-- theme: info -->

> > These instructions have been tested on a **MacBook Air** with a M1 chip.

Reviewer: ```suggestion
> These instructions have been tested on a **MacBook Air** with an M1 chip.

```

</div>
</div>
</div>

> Open terminal using Rosetta.

Reviewer: Should these be numbered lists since they are sequential?

Reviewer 2: Bumping this. Should the line above also be numbered?

# Install Stencil CLI
npm install -g @bigcommerce/stencil-cli
```

### Installing on Mac with Apple silicon

Installing Stencil CLI and its dependencies on Macs that use Apple silicon, such as the M1 chip, requires Rosetta. Rosetta allows a Mac with Apple silicon to use apps built for a Mac with an Intel processor. The following steps will guide you through the installation process:

<div class="HubBlock--callout">
<div class="CalloutBlock--info">
<div class="HubBlock-content">

<!-- theme: info -->

> These instructions have been tested on a **MacBook Air** with an M1 chip.

</div>
</div>
</div>

Open terminal using Rosetta.
1. Open **Finder**.
2. Go to **Applications** > **Utilities** > **Terminal**.
3. Right-click **Terminal** and select **Get Info**.
4. Check the **Open using Rosetta** checkbox.
5. Close the window and quit all terminal instances.
6. Start a new terminal, and install Rosetta if prompted.

> Run the following commands.

Reviewer 2: Is this also a step in the process? Should it be numbered?

Author: This process is consistent with the other ones.

Author: The steps are consistent with the commands for the other platforms (Windows, Linux).
